### PR TITLE
Panel Border Around Facets

### PR DESCRIPTION
**Pull request by DarioS**
Makes plot more interpretable when there are five or so factor levels. This would solve #1491.

### CODE REVIEW

File: R/theme-defaults.r [hunk 1]
```diff
@@ -227,7 +227,7 @@ theme_minimal <- function(base_size = 12, base_family = "") {
 theme_classic <- function(base_size = 12, base_family = ""){
   theme_bw(base_size = base_size, base_family = base_family) %+replace%
     theme(
-      panel.border     = element_blank(),
+      panel.border     = element_rect(fill = "transparent"),
```
Comment (hadley) [inline R/theme-defaults.r]: Can you please use `fill = NA` to be consistent?

File: R/theme-defaults.r [hunk 2]
```diff
@@ -275,3 +275,5 @@ theme_void <- function(base_size = 12, base_family = "") {
   )
 }
 
+, size = 0.125),
```
Comment (hadley) [inline R/theme-defaults.r]: Was this included by accident?

Comment (DarioS) [inline R/theme-defaults.r]: Yes.

**Comment by hadley** [comment]
This totally changes the non-facetted appearance, which I think is the most common use case.

**Comment by DarioS** [comment]
It seems impossible to make the theme look nice for both use cases, without user customisation at runtime.

**Comment by hadley** [comment]
In that case, I think its best to stick with current behaviour

**Pull request closed without merging**